source/hardware/controllers/native_instruments_traktor_kontrol_s4_mk2.rst

```diff
@@ -71,11 +73,14 @@ Control                                             Description
 ==================================================  =========================================================================
 :hwlabel:`LOAD`                                     Load track selected in library to the deck.
 :hwlabel:`SHIFT` + :hwlabel:`LOAD`                  Eject track
-Small buttons with play icons                       Play a sampler from its cue point. If no track is loaded in the sampler, the track selected in the library will be loaded.
-:hwlabel:`SHIFT` + small buttons with play icons    If sampler is playing, stop it. If sampler is not playing, the loaded eject track from the sampler.
+Small buttons with preview icons                    Play a sampler from its cue point. If no track is loaded in the sampler, the track selected in the library will be loaded.
+:hwlabel:`SHIFT` + small buttons with preview icons If sampler is playing, stop it. If sampler is not playing, the loaded track is ejected from the sampler.
 :hwlabel:`1-4` numbered buttons                     Set/activate :term:`hotcue`
 :hwlabel:`SHIFT` + :hwlabel:`1-4` numbered buttons  Clear hotcue
 :hwlabel:`SHIFT` + Wheel nudge                      Fast search through track when not playing
+:hwlabel:`FLUX` button                              Enable slip mode (if shift is held down this decreases the range of the BPM slider, is play button is held down enter autoslip mode)
```

Comment:
```suggestion
:hwlabel:`FLUX` button                              Enable slip mode (if shift is held down this decreases the range of the BPM slider, if play button is held down enter autoslip mode)
```
What is autoslip mode? Isn't it problematic to need to hold down the play button for this because it would also start/stop the deck?

Reply:
```suggestion
:hwlabel:`FLUX` button                              Enable slip mode. If :hwlabel:`SHIFT` is held down this decreases the range of the BPM slider. If the :hwlabel:`PLAY` button is held down enter autoslip mode. 
```

Reply:
This is a bit confusing. If you look at the center of the mixer you see there's a button annoyingly labelled 'Play'
![image](https://user-images.githubusercontent.com/4432814/99892569-f21d5c80-2c6d-11eb-85bf-a238c759e654.png)

Reply:
You can distinguish these by using something like:
- :hwlabel:`PLAY` (:hwlabel:`TRANSPORT` section)
- :hwlabel:`LOOP RECORDER PLAY`